services/core-web/src/components/modalContent/DownloadDocumentPackageModal.js

```diff
@@ -126,6 +126,7 @@ export const DownloadDocumentPackageModal = (props) => {
           <NOWActionWrapper
             permission={Permission.EDIT_PERMITS}
             tab={props.type === "FNC" ? "CON" : props.type}
+            isDisabledReviewButton={true}
           >
```

Comment:
You don't need to specify true, the existence of the prop is enough.

Reply:
Right, removing it.